src/frontend/js/utils/api/web-analytics/index.ts

```diff
@@ -0,0 +1,18 @@
+import { WebAnalyticsAPI, WebAnalyticsAPIBackend } from 'types/WebAnalytics';
+import context from 'utils/context';
+import GoogleAnalyticsApi from './google_analytics';
+import GoogleTagManagerApi from './google_tag_manager';
+
+const WEB_ANALYTICS_PROVIDER = context?.web_analytics_provider;
+
+const WebAnalyticsAPIHandler = (): WebAnalyticsAPI | undefined => {
```

Comment:
We have a type `Maybe` for this kind of case.

```suggestion
const WebAnalyticsAPIHandler = (): Maybe<WebAnalyticsAPI> => {
```